db/seeds.rb

```diff
@@ -7,8 +7,18 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 User.create(username: "test", email: "test@m.com", password: "12345678")
 
-energy = Title.create(name: "Energy use (kg of oil equivalent per capita)")
-biodiversity = Title.create(name: "GEF benefits index for biodiversity")
+Title.create([
+  { value: "AG.LND.FRST.ZS", name: "Forest area (% of land area)" },
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.